sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_metrics_advisor_client.py

```diff
@@ -401,25 +431,40 @@ def list_anomalies_for_detection_configuration(self, detection_configuration_id,
         :raises ~azure.core.exceptions.HttpResponseError:
         """
 
-        skip = kwargs.pop('skip', None)
-        filter_condition = kwargs.pop('filter', None)
-        converted_start_time = convert_datetime(start_time)
-        converted_end_time = convert_datetime(end_time)
-        detection_anomaly_result_query = DetectionAnomalyResultQuery(
-            start_time=converted_start_time,
-            end_time=converted_end_time,
-            filter=filter_condition,
-        )
-
-        return self._client.get_anomalies_by_anomaly_detection_configuration(  # type: ignore
-            configuration_id=detection_configuration_id,
-            skip=skip,
-            body=detection_anomaly_result_query,
-            cls=lambda objs: [DataPointAnomaly._from_generated(x) for x in objs],
-            **kwargs)
+    @distributed_trace
+    def list_anomalies(self, **kwargs):
+        # type: (Any) -> ItemPaged[DataPointAnomaly]
+        """Query anomalies under a specific alert or detection configuration.
```

Comment:
Looking at rendered documentation here, I think we'll need to include the filter kwarg for the detection config flavor of the method since it doesn't show up.

        :keyword filter:
        :paramtype filter: ~azure.ai.metricsadvisor.models.DetectionAnomalyFilterCondition

Also thinking that in the short term, maybe we should specify in the docstring which sets of keywords can be passed together.


![image](https://user-images.githubusercontent.com/31998003/97616453-d1991400-19d9-11eb-8eb0-59512ab3f31a.png)